.github/dependabot.yml

```diff
@@ -50,6 +43,10 @@ updates:
       OpenTelemetry:
         patterns:
           - "OpenTelemetry.*"
+      NetPlatform:
+        patterns:
+          - "Microsoft.AspNetCore.*"
+          - "Microsoft.Extensions.*"
```

Comment:
Add "Microsoft.Orleans*" as well?

also feel free to replace/update https://github.com/dotnet/aspire-samples/pull/369

Reply:
> Add "Microsoft.Orleans*" as well?

Does it ship at the same time on patch Tuesdays? Or do you mean create a new group for Orleans?